tests/test_views.py

```diff
@@ -24,13 +24,88 @@
 
 class ViewsTests(InvenioTestCase):
     """Tests views."""
+    def setUp(self):
+        self.login('admin', 'admin')
```

Comment:
I suspect the login/logout part may slow down the tests a bit, so if you do not need it for every test then we can just call it in the few tests that do.